src/Main.java

```diff
@@ -1,4 +1,78 @@
-package PACKAGE_NAME;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class Main {
+
+
+    public static int countOccurrence(List<String> list, String word) {
+        int count = 0;
+        for (String s : list) {
+            if (s.equals(word)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(int[] array) {
+        List<Integer> list = new ArrayList<>();
+        for (int value : array) {
+            list.add(value);
+        }
+        return list;
+    }
+
+
+    public static List<Integer> findUnique(List<Integer> list) {
+        return new ArrayList<>(new HashSet<>(list));
```

Comment:
all code that contains Map or Set needs to be rewritten without using it. We don't use what we have not learned